app/Models/Camp.php

```diff
@@ -453,7 +453,11 @@ public static function getDirectCampSupporterIds($topicid, $campnum)
     public static function validateParentsupport($topicNum, $campNum, $userNicknames) 
     {
         $filter = self::getLiveCampFilter($topicNum, $campNum);
-        $oneCamp = self::getLiveCamp($filter);        
+        $oneCamp = self::getLiveCamp($filter);   
+
+        if(empty($oneCamp)){
+            return 'notfound';
```

Comment:
May be return 0 or false instead of string?

Reply:
Zero or false will not work as False or Zero  means no support, but its the case of not found record.